src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4314,8 +4340,8 @@ .FORWARDHELPCATEGORY Cmdlet
             if ($pagerArgs) {
                 # Supply pager arguments to an application without any PowerShell parsing of the arguments.
                 # Leave environment variable to help user debug arguments supplied in $env:PAGER.
-                $env:__PSPAGER_ARGS = $pagerArgs
-                $help | Out-String -Stream -Width ($consoleWidth - 1) | & $pagerCommand --% %__PSPAGER_ARGS%
+                # $env:__PSPAGER_ARGS = $pagerArgs
```

Comment:
should probably wrap this with check for experimental feature so we can delete the env var when this feature is stable

Reply:
fixed